L1Trigger/VertexFinder/python/VertexProducer_cff.py

```diff
@@ -78,7 +78,7 @@
         PVZ0Graph = cms.string("L1Trigger/VertexFinder/data/z0Corr/Quantised_model_prune_iteration_9_patternModelgraph.pb"),
         # Adhoc correction to track z0 to correct for upstream asymmetry in tracks:
         apply_z0Correction = cms.bool(True),
-        z0Correction = cms.double(-0.03)
+        z0Correction = cms.double(0.03)
```

Comment:
Is this to correct the effect I saw in my plots?

Reply:
Yes, though it would be useful to have 3 comparisons, one without the correction, one with -0.03 and one with +0.03